code/modules/mob/living/living.dm

```diff
@@ -846,6 +846,9 @@
 			lying = incapacitated(INCAPACITATION_KNOCKDOWN)
 			canmove = !incapacitated(INCAPACITATION_DISABLED)
 
+	if(leashed) // ARFS ADD: LEASHES BOIIIIIIIIIIIIIIIIIIIII
```

Comment:
I know we really don't want to be touching base files much. @TalkingCactus Thoughts?

Reply:
DO NOT TOUCH BASE FILES

Reply:
Only way I can implement it this way. I even said in voice.